backend/src/api/mutation.rs

```diff
@@ -232,12 +232,20 @@ impl Mutation {
         BlockValue::remove(id, context).await
     }
 
-    /// Atomically mount a series into an (empty) realm.
+    /// Atomically mount or move a series into an (empty) realm.
     /// Creates all the necessary realms on the path to the target
     /// and adds a block with the given series at the leaf.
+    /// 
+    /// If the series is already mounted in `current_realm_path`, we need to check
+    /// if that realm has any other blocks.
+    /// - If it does, moving the series is not allowed.
+    /// - Otherwise, the series is moved, but there is some cleanup necessary:
+    ///   - If the realm has any sub-realms: only the series block is removed.
+    ///   - Otherwise the whole realm is removed.
     async fn mount_series(
         series: NewSeries,
         parent_realm_path: String,
+        current_realm_path: Option<String>,
         #[graphql(default = vec![])]
         new_realms: Vec<RealmSpecifier>,
         context: &Context,
```

Comment:
I would create a separate endpoint for this functionality. `mount_series` is already way too overloaded and does too many things. Maybe `move_mounted_series` or sth.

Or actually... why not just add `remove_series_mount`? Then the Admin UI can call that, followed by another `mount_series`. Is there any advantage in having the "delete" and "create" in the same endpoint? I can imagine that in the future, people want to be able to just remove a mount in the admin UI, without moving it somewhere else.